src/themes/creator.js

```diff
@@ -572,7 +572,16 @@ export default function createTheme(
       radius200: '4px',
       radius300: '8px',
       radius400: '12px',
+      /** Checkbox, Datepicker (Range), Progress Bar, Slider, Tag */
       useRoundedCorners: true,
+      /** Button, ButtonGroup */
+      buttonBorderRadius: '0px',
+      /** Input, Select, Textarea */
+      inputBorderRadius: '0px',
+      /** Popover, Menu, Tooltip */
+      popoverBorderRadius: '0px',
+      /** Card, Datepicker, Modal, Toast, Notification */
+      defaultBorderRadius: '0px',
```

Comment:
`surfaceBorderRadius`? Why does popover need to have a different value than card?

Reply:
or, directly use a radius value rather than abstract through a semantic name

Reply:
Popover, Menu, and Tooltip all used to have 8px corners, so for people coming from <=v6 they would need those grouped.

Reply:
that makes sense, 'default' seems to convey the wrong concept - it's not as if this will change every usage of border radius

Reply:
It should convey the basic shape/silhouette that most UI elements will have.

`baseBorderRadius`
`basicBorderRadius`
`commonBorderRadius`
`containerBorderRadius`

Reply:
I didn't notice the `surfaceBorderRadius` suggestion - I think we can go with that. Not sure there is a perfect semantic grouping for these left over components.

Honestly we probably only need 2 variables: `surfaceBorderRadius` and `popoverBorderRadius` 🤔... but for now this will be fine - can revisit this in an upcoming major when we finish the design migration.